src/hooks/useFetchWalletTokens.ts

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from "react";
```

Comment:
These hooks seem very useful to be able to pull into another module that can be used outside of just the creator-ui. What do you think about doing a sol-kit-react module? Or some structure along those lines? Maybe even including in sol-kit, but that seems like forcing more opinionated usage of React into sol-kit. I meant sol-kit to be as un-opinionated as possible. Thoughts?

Reply:
Agree with keeping sol-kit unopinionated. We could do a separate sol-kit/react bundle?

Reply:
@long-banana I had a go at this today using yarn workspaces on the sol-kit side with sol-kit and sol-kit-react as the packages. I'm running into yarn issues trying to test locally though (link doesn't work for unpublished packages) - any tips?

Reply:
If you push up what you have I can take a look into it further. I have some other pressing things today, but I should be able to find some time for it. 

Are you saying you’re using the sol-kit repo as a mono repo for both packages? So I don’t need to make a new repo for you to push?

Reply:
Yeah that's the idea. I'll give it another crack in the morning and push something up for you to check out.

Reply:
@long-banana https://github.com/raindrop-studios/sol-kit/compare/feature/sol-kit-react?expand=1

Reply:
@long-banana I'm going to merge this one for now, and pull the utils out as a separate task